endOfEpoch/shardchain/trigger.go

```diff
@@ -35,12 +35,13 @@ type ArgsShardEndOfEpochTrigger struct {
 }
 
 type trigger struct {
-	epoch             uint32
```

Comment:
Check namings end of epoch -> epoch start

Reply:
done